kubernetes/linux/main.sh

```diff
@@ -825,6 +825,10 @@ if [ "${CONTAINER_TYPE}" != "PrometheusSidecar" ] && isGenevaMode; then
     echo "export MDSD_FLUENT_SOCKET_PORT=$MDSD_FLUENT_SOCKET_PORT" >> ~/.bashrc
     export SSL_CERT_FILE="/etc/pki/tls/certs/ca-bundle.crt"
     echo "export SSL_CERT_FILE=$SSL_CERT_FILE" >> ~/.bashrc
+    if [ "${USING_AAD_MSI_AUTH}" == "true" ]; then
```

Comment:
looks like both this parent if and else block have this check?

Reply:
> looks like both this parent if and else block have this check?

No, parent if doesnt have this check. This is for geneva path only.